docs/data-sources/identity_group.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Identity and Access Management (IAM)"
+---
+
+# huaweicloud_identity_group
+
+Use this data source to get the ID of an IAM **User Group**.
```

Comment:
Does UserGroup need to be bold? Many other titles of data source are not in bold.

Reply:
fixed